api/src/main/java/org/openmrs/module/fhir/api/util/FHIRObsUtil.java

```diff
@@ -240,4 +243,47 @@ public static Observation generateObs(Obs obs) {
 		}
 		return observation;
 	}
+	
+	public static Obs F(Observation observation) {
+		Obs obs = new Obs();
+		
+		ResourceReferenceDt subjectref = observation.getSubject(); // subject can be null, handle that, and abort
+		IdDt id = subjectref.getReference();
+		String uri = id.getValue();
+		String[] paths = uri.split("/");
+		String patientUuid = paths[1];
+		obs.setPerson(Context.getPersonService().getPersonByUuid(patientUuid));
+		
+		DateTimeDt dateApplies = (DateTimeDt) observation.getApplies();
+		obs.setObsDatetime(dateApplies.getValue());
+		
+		Date instant = observation.getIssued();
+		obs.setDateCreated(instant);
+		
+		CodeableConceptDt dt = observation.getCode();
+		List<CodingDt> dts = dt.getCoding();
+		CodingDt coding = dts.get(0);
```

Comment:
@surangak can you put a link on  where the specification says that the code is always will be there?